Makefile

```diff
@@ -22,4 +22,4 @@ mypy:
 	pipenv run mypy transmogrifier
 
 test:
-	pipenv run pytest --cov=transmogrifier
\ No newline at end of file
+	pipenv run pytest --cov-report term-missing --cov=transmogrifier
```

Comment:
I also propose we modify our usual `make test` command to include the uncovered lines, I can update the dev docs if others agree